shared/utils.js

```diff
@@ -3,7 +3,12 @@
 // TODO: make all of these functions configurable from the outside
 // the user should be able to manipulate those
 
-function getNamespace(service, stage) {
+function getNamespace(serverless, stage) {
```

Comment:
I think we can simplify it to just take `serverless`, and `stage` can be resolved out of `serverless` via `serverless.service.provider.stage`

Reply:
good point, done!